presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +159,10 @@ trap terminate INT TERM EXIT
 # start hadoop container
 docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d hadoop-master
 
+# start external database containers
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d mysql
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d postgres
```

Comment:
@maciejgrzybek RE: "Shouldn't hadoop-kerberized be a base for all kerberized dockers?" Good idea.  I will re-work the tree so that there is a kerberized-linux base image.